ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml

```diff
@@ -699,7 +699,7 @@ periodics:
       - name: OCM_CCS
         value: "true"
       - name: SECRET_LOCATIONS
-        value: /usr/local/osde2e-common,/usr/local/osde2e-credentials,/usr/local/smart-events-osde2e-secret
```

Comment:
osde2e-common and osde2e-credentials contain OSDe2e's integration secrets but not CCS credentials.

Reply:
last PROW job is failing because its not able to locate our the secret(`smart-events-osde2e-secret`). Where should I configure this secret? What value should I pass to it?

Reply:
Replied in CoreOS #SD-CICD channel
Secret setup can be found following this doc:
https://github.com/openshift/osde2e/blob/main/docs/Addons.md